Presentation.Web/Controllers/API/V1/KendoOrganizationalConfigurationController.cs

```diff
@@ -43,25 +52,41 @@ public HttpResponseMessage SaveConfiguration([FromBody] KendoOrganizationalConfi
         [SwaggerResponse(HttpStatusCode.Created, Type = typeof(ApiReturnDTO<KendoOrganizationalConfigurationDTO>))]
         [SwaggerResponse(HttpStatusCode.BadRequest)]
         [SwaggerResponse(HttpStatusCode.NotFound)]
-        public HttpResponseMessage GetConfiguration([FromUri] int? organizationId, [FromUri] OverviewType? overviewType)
+        public HttpResponseMessage GetConfiguration([FromUri][Required] int? organizationId, [FromUri][Required] OverviewType? overviewType)
         {
-            if (organizationId == null || overviewType == null)
-                return BadRequest("Please provide both organizationId and overviewType");
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
 
             return _kendoOrganizationalConfigurationService
                 .Get(organizationId.Value, overviewType.Value)
                 .Match(value => Ok(Map(value)), FromOperationError);
         }
 
+        [HttpGet]
+        [Route("version")]
+        [SwaggerResponse(HttpStatusCode.Created, Type = typeof(ApiReturnDTO<string>))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage GetConfigurationVersion([FromUri][Required] int? organizationId, [FromUri][Required] OverviewType? overviewType)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            return _kendoOrganizationalConfigurationService
+                .Get(organizationId.Value, overviewType.Value)
+                .Select(x => x.Version)
+                .Match(Ok, FromOperationError);
+        }
+
         [HttpDelete]
         [Route]
         [SwaggerResponse(HttpStatusCode.Created, Type = typeof(ApiReturnDTO<KendoOrganizationalConfigurationDTO>))]
         [SwaggerResponse(HttpStatusCode.BadRequest)]
         [SwaggerResponse(HttpStatusCode.NotFound)]
-        public HttpResponseMessage DeleteConfiguration([FromUri] int? organizationId, [FromUri] OverviewType? overviewType)
+        public HttpResponseMessage DeleteConfiguration([FromUri][Required] int? organizationId, [FromUri][Required] OverviewType? overviewType)
```

Comment:
må de gerne være null? for null er jo egentlig gyldigt for int? hvis blot der f.eks, står organizationId=null i query?